examples/vuejs-live-cursors/README.md

```diff
@@ -48,9 +45,11 @@ Alternatively, you can set up your project manually:
 
 - Install all dependencies with `npm install`
 - Create an account on [liveblocks.io](https://liveblocks.io/dashboard)
-- Copy your **public** key from the [dashboard](https://liveblocks.io/dashboard/apikeys)
-- Replace `pk_YOUR_PUBLIC_KEY` in [`src/App.vue`](./src/App.vue) with your **public** key
-- Run `npm run serve` and open `http://localhost:8000` in your browser
+- Copy your **public** key from the
+  [dashboard](https://liveblocks.io/dashboard/apikeys)
+- Create an `.env.local` file and add your **public** key as the
+  `VITE_LIVEBLOCKS_PUBLIC_KEY` environment variable
+- Run `npm run dev` and go to [http://localhost:5173](http://localhost:5173)
```

Comment:
Why port 5173?

Reply:
It's the [Vite default](https://vitejs.dev/config/server-options.html#server-port)